Add basic infos about Gamify-IT (#1)
Gamify-IT/.github: profile/README.md

### Finitequiz Backend

The [finitequiz backend repo](https://github.com/Gamify-IT/finitequiz-backend) stores the Finitequiz data, like minigame runs.

### Git Card Game

The [git card game repo](https://github.com/Gamify-IT/git-card-game) contains the Vue/Typescript Code for the minigame Website Git Card Game.
This minigame is a prototype.

### Git Card Game Backend

The [git card game backend repo](https://github.com/Gamify-IT/git-card-game-backend) stores the Git Card Game data, like minigame runs.

### Regex Game

The [regex game repo](https://github.com/Gamify-IT/regex-game) contains the Vue/Typescript Code for the minigame Website Regex Game.
This minigame is a prototype.

### Actions Runner

The [actions runner repo](https://github.com/Gamify-IT/actions-runner)

### Authentification Validator

The [authentification validator repo](https://github.com/Gamify-IT/authentification-validator)

### Issues

The [issues repo](https://github.com/Gamify-IT/issues) contains all the issues across the Project. Instead of opening issues in the individual repos, they are created here.

### Landing Page

The [landing page repo](https://github.com/Gamify-IT/landing-page)

### Run config

> The [run config repo](https://github.com/Gamify-IT/run-config)

Reviewer: :laughing: I'll accept it for now.

> # Gamify-IT Quickstart Guide

Reviewer: ```suggestion
# Gamify-IT
```

# Gamify-IT Quickstart Guide

> Gamify-IT is a minigame platform for students. With Gamification the students are able to learn lecture content through variable minigames. There is a world where the student can walk through and interact with.  

Reviewer: ```suggestion
_Gamify-IT_ is a minigame platform for students.  
Students are able to learn content by playing minigames.  
Additionally, they can explore the large overworld and with interact it.  
```

> A lecturer sets up the World and the minigames. He is able to add Questions about lecture content to the minigames, add hints for the students, etc.  
> There is a progress system for the students to unlock new content.

Reviewer: ```suggestion
A lecturer configures the world and minigames.  
They determine what questions the students are asked in a minigame, what hints to give the students (i.e. when a student talks to an NPC), … 
```

## Overview

> ### world

Reviewer: ```suggestion
### World
```

![World](https://raw.githubusercontent.com/Gamify-IT/docs/main/images/overworld.webp)

> The Player (in the middle of the picture) is able to move around the World and interact with variable content.

Reviewer: ```suggestion
The player (in the middle of the picture) is able to move around the World and interact with customizable content.
```

### Lecturer Interface

![Lecturer Interface](https://raw.githubusercontent.com/Gamify-IT/docs/main/images/lecturer-interface.webp)

> The Lecturer has access to an interface where he can configure the worlds (for example add questions regarding the topic of the lecture)

Reviewer: ```suggestion
The lecturer can configure everything with the [lecturer interface](https://github.com/Gamify-IT/lecturer-interface).  
In the future, they might also see course statistics there.
```

## Repositories

information about all the different repositories

> ### [Docs](https://github.com/Gamify-IT/docs)

Reviewer: ```suggestion
### Docs
```

> The Docs repository contains the documentation about the minigame platform

Reviewer: ```suggestion
The [docs repo](https://github.com/Gamify-IT/docs) contains the complete documentation.  
We recommend <https://gamifyit-docs.readthedocs.io/en/latest/> to view the docs as it always displays the content in a visually enhanced way.
```

> ### [Overworld](https://github.com/Gamify-IT/overworld)
> 
> The Overworld repository contains the Unity Project on which the World is build with. It contains all the Assets, Scripts, Game Logic, etc. that is needed for the World to run.

Reviewer: ```suggestion
### Overworld

The [overworld repo](https://github.com/Gamify-IT/overworld) contains all the assets, scripts, game logic, … to display and walk through the world.
```

> ### [Overworld-Backend](https://github.com/Gamify-IT/overworld-backend)
> 
> The Overworld-Backend repository contains the Java backend of the World. All relevant data of the World is stored in the Overworld-Backend

Reviewer: ```suggestion
### Overworld Backend

The [overworld backend repo](https://github.com/Gamify-IT/overworld-backend) stores all overworld data, whether it be the content of a course or player statistics.  
Minigame runs are stored in the backend of the respective minigame.
```
